spec/factories/lead_provider.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :lead_provider do
-    name  { "Lead Provider" }
+    name  { Faker::Name.name }
```

Comment:
I reckon this will cause a lot of randomness to show up in percy snapshots :/

Reply:
Reverted